### ISSUE TITLE
Add Multiple Items batch only work for more than 4 items

### ISSUE THREAD

**Issue by JackHu88** [SharePoint/PnP-JS-Core#392]
#### Category
[ ] Enhancement

[ x] Bug

[ ] Question

#### Expected / Desired Behavior / Question
I want to add multiple items to a list, but when items less than 4 then batch can't work.

#### Steps to Reproduce
Below code can't work, when i change the **i** large than 3, for example 4, then it can work.

```
var list = $pnp.sp.web.lists.getByTitle("BatchTest");

list.getListItemEntityTypeFullName().then(entityTypeFullName => {

    var batch = $pnp.sp.web.createBatch();

    for(i=0;i<3;i++){
	    list.items.inBatch(batch).add({ Title: "Batch"+i }, entityTypeFullName).then(b => {
            console.log(b);
        });
    }    
    batch.execute().then(d => console.log("Done"));
});

```

**Comment by patrick-rodgers** [comment]
What version are you using for this test?

**Comment by JackHu88** [comment]
I'm using the latest pnp.js rebuilt from dev branch

**Comment by JackHu88** [comment]
Hi @patrick-rodgers , did you find the problem?

**Comment by patrick-rodgers** [comment]
Hi @JackHu88 - this is fixed and will be part of the 2.0.3 release. If you find an issue that is critical to your work you can always debug, update, and submit a PR to resolve it yourself. This will ensure it is fixed for an upcoming release in case the core team members do not have time to look at it. We are all volunteers supporting this effort.

Fixed in PR #408

**Comment by JackHu88** [comment]
Thanks, i will try :)
